### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by stephly07**
Removed unnecessary punctuation marks and words.
Moved text to a Notes section since it looked like it should be called out that way.

### CODE REVIEW

File: guides/user/pipeline/triggers/github/index.md [hunk 1]
```diff
@@ -16,18 +16,18 @@ files as [artifacts](/reference/artifacts) into a pipeline.
...
 
-If you (or your Spinnaker admin) have already configured Spinnaker to listen to
+If you (or your Spinnaker admin) already configured Spinnaker to listen to
 a GitHub webhooks from the repository you plan to publish commits to, you can
 skip this section.
 
 You need the following:
 
-* A GitHub repository either under your user, or in an organization or user's
+* A GitHub repository either under your user, in an organization, or user's
```
Comment (dorbin) [inline guides/user/pipeline/triggers/github/index.md]:
```suggestion
* A GitHub repository, either under your user account or in an organization or user's
```

Comment (dorbin) [reply guides/user/pipeline/triggers/github/index.md]: I think we need the "or" here. I took another shot at this one.

Comment (stephly07) [reply guides/user/pipeline/triggers/github/index.md]: Okay that's fine!

Comment (dnilasor) [reply guides/user/pipeline/triggers/github/index.md]: +1 @dorbin

**Review by dnilasor — CHANGES_REQUESTED** [review]
@stephly07, you can use the "Accept Changes" (pretty sure that is what it's called) button to accept @dorbin's suggestions into your fork/PR. Once you've done that, approved

**Review by dorbin — APPROVED** [review]

**Pull request merged**